src/sentry/relay/config/__init__.py

```diff
@@ -47,8 +47,6 @@
 from .measurements import CUSTOM_MEASUREMENT_LIMIT
 
 # These features will be listed in the project config.
-#
-# NOTE: These features must be sorted or the tests will fail!
```

Comment:
This is still the case.

Reply:
Unless there is more context, it doesn't appear as if this list is currently sorted, at least not alphabetically, so I figured I'd test that comment assertion.

Can you please link the librelay build you mentioned?

Reply:
Ah I guess the build succeeded, maybe the test changed, so nothing to do. But eventually you will have to add it to [Relay](https://github.com/getsentry/relay/blob/cc7242a9f8f8327ec740a68827c81d089940735b/relay-dynamic-config/src/feature.rs#L13-L122).

Librelay, is what we call the python library built from Relay, it's pulled into Sentry as [`sentry-relay`](https://github.com/getsentry/sentry/blob/master/requirements-frozen.txt#L133).

Reply:
Yeah since it was already unordered I suspected that comment wasn't true so I figured it be better to not warn on it. 

I've already added the feature to relay in a PR I've yet to put up, just came to do this first since usually serialization should happen before consuming.

This seems fine to merge in that case?

Reply:
Yeah it's fine, Relay can deal with it either way, I just expected the tests to be failing (that used to be the case) :shipit: